.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -8,11 +8,22 @@ assignees: ''
 ---
 
 **Concisely describe the proposed feature**
-A clear and concise description of what you want. For example, 
-> I would like to add an Apple Metal backend to the compiler so that my Macbook GPU can be utilized.
+A clear and concise description of what you want. For example:
+
+I would like to add an Apple Metal backend to the compiler so that my Macbook GPU can be utilized.
+
+**Think about the advantages and disadvantages of the proposed feature**
+All coins have its two side, we encourage people to think critically. For example:
+
+Advantages:
+1. So that Apple users could utilize their GPU resources.
+
+Disadvantages:
+1. Increases the maintainance cost over multiple backends.
```

Comment:
The older version actually looks better. It's good to think about "two sides", but we don't want to overwhelm newcomers.

Reply:
How about `Think about the advantages and disadvantages of the proposed feature (if any)`? For example I immediately closed #1648 after thought about not-return-by-reference. This is actually helpful for people to rethink the reason why we need to add this feature and search for better solutions.